IBAnimatable/PresentExplodeWithDismissInteractionSegue.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 
 public class PresentExplodeWithDismissInteractionSegue: UIStoryboardSegue {
   public override func perform() {
-    destinationViewController.transitioningDelegate = PresenterManager.sharedManager().retrievePresenter(.Explode)
+    destinationViewController.transitioningDelegate = PresenterManager.sharedManager().retrievePresenter(.Explode, interactiveGestureType: .Default)
     sourceViewController.presentViewController(destinationViewController, animated: true, completion: nil)
   }
 }
```

Comment:
we can remove it now and add it back later when we have pinch gesture and want to use pinch to dismiss Explode animation.